packages/ui-text-input/src/TextInput/styles.ts

```diff
@@ -116,8 +106,10 @@ const generateStyle = (
     outline: 'none',
     fontFamily: componentTheme.fontFamily,
     fontWeight: componentTheme.fontWeight,
-    color: componentTheme.color,
-    padding: `0 ${componentTheme.padding}`,
+    // padding of the text in the input
+    padding: `0 ${componentTheme.gapContent} 0  ${
+      paddingHorizontalVariants[size!]
```

Comment:
It should be `${componentTheme.gapContent}` here as well for the left padding

Reply:
fixed